src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1519,7 +1551,12 @@ public function insert_posts( $items = [] ) {
 			if ( ! empty( $event['Venue']['Venue'] ) ) {
 				$event['Venue']['Venue'] = trim( $event['Venue']['Venue'] );
 
-				if ( ! empty( $item->venue->global_id ) || in_array( $this->origin, array( 'ics', 'csv', 'gcal', 'ical' ) ) ) {
+				if ( ! empty( $item->venue->global_id ) || in_array( $this->origin, [
+						'ics',
+						'csv',
+						'gcal',
+						'ical',
+					] ) ) {
```

Comment:
Save this into a variable, this style of line breaking makes it hard to read.

Reply:
Great point :+1: